src/modules/test/MimParser.cpp

```diff
@@ -0,0 +1,65 @@
+#include "MimParser.h"
+
+#include <cstdio>
+#include <string.h>
+#include <parson.h>
```

Comment:
Consider creating a new Common.h or similar header that has all includes and then include that everywhere.